code/API_definitions/kyc-match.yaml

```
@@ -0,0 +1,536 @@
+
+        postalCodeMatch:
+          type: string
+          description: Indicates whether the postal code of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        regionMatch:
+          type: string
+          description: Indicates whether the region of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        localityMatch:
+          type: string
+          description: Indicates whether the locality of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        countryMatch:
+          type: string
+          description: Indicates whether the country of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        houseNumberExtensionMatch:
+          type: string
+          description: Indicates whether the house number extension of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        birthdateMatch:
+          type: string
+          description: Indicates whether the birthdate of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        emailMatch:
+          type: string
+          description: Indicates whether the email address of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+        genderMatch:
+          type: string
+          description: Indicates whether the gender of the customer matches with the one on the OB systems.
+          enum:
+                - 'true'
+                - 'false'
+                - 'not_available'
+
+    ErrorInfo:
+      type: object
+      required:
+        - status
+        - code
+        - message
+      properties:
+        status:
+          type: integer
+          description: HTTP response status code
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+
+  responses:
+    Generic400:
+      description: |-
+        Problem with the client request.
+        In addition to regular scenario of `INVALID_ARGUMENT`, another scenario may exist.
+
+        - Indicated param combination is invalid (`"code": "KNOW_YOUR_CUSTOMER.INVALID_PARAM_COMBINATION","message": "Indicated parameter combination is invalid"`)
+      content:
+        application/json;charset=UTF-8:

```

According to CAMARA guidelines:

```suggestion
        application/json:

```